mathics/builtin/comparison.py

```diff
@@ -349,17 +386,42 @@ def apply_other(self, x, y, evaluation):
 
 
 class Less(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'Less[$x$, $y$]'
+    <dt>'$x$ < $y$'
+        <dd>yields 'True' if $x$ is known to be less than $y$.
+    <dt>'$lhs$ < $rhs$'
+        <dd>represents the inequality $lhs$ < $rhs$.
+    </dl>
+    """
     operator = '<'
     sympy_name = 'StrictLessThan'
 
 
 class LessEqual(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'LessEqual[$x$, $y$]'
+    <dt>'$x$ <= $y$'
+        <dd>yields 'True' if $x$ is known to be less than or equal to $y$.
+    <dt>'$lhs$ <= $rhs$'
+        <dd>represents the inequality $lhs$ ≤ $rhs$.
```

Comment:
I can see the benefit of full TeX support, but I'm not sure the best way to handle the conversions. In principle we want to be able to convert to latex (for docs), html (for online docs) and plaintext (for introspection, e.g. `Information`). We could probably leverage pandoc to do the conversions. Is there a better way?